setup.py

```diff
@@ -34,7 +34,8 @@
       url="http://bioblend.readthedocs.org/",
       install_requires=['requests>=2.4.3,!=2.12.0,!=2.12.1', 'requests-toolbelt', 'boto>=2.9.7', 'pyyaml', 'six'],
       tests_require=tests_require,
-      packages=find_packages(exclude=['tests']),
+      packages=find_packages(),
```

Comment:
Is this first change actually needed? `tests/` directory is needed only for `run_bioblend_tests.sh`, which is supposed to be a script for developers.